src/Sulu/Component/Content/Mapper/ContentMapperInterface.php

```diff
@@ -302,13 +302,6 @@ public function orderBefore($uuid, $beforeUuid, $userId, $webspaceKey, $language
     public function orderAt($uuid, $position, $userId, $webspaceKey, $languageCode);
 
     /**
-     * TRUE ignores mandatory in save
-     * @param bool $ignoreMandatoryFlag
-     * @return $this
-     */
-    public function setIgnoreMandatoryFlag($ignoreMandatoryFlag);
```

Comment:
yeah i think it was used in a early stadium of preview! where we use the content mapper to save the temporary content!